operations/companies.md

```diff
@@ -236,12 +242,22 @@ Adds a new company to the enterprise.
 | `Contact` | string | optional | Other contact details, such as telephone, email or similar. |
 | `Notes` | string | optional | Notes of the company. |
 | `Iata` | string | optional | Iata of the company. |
+| `Department` | string | optional | The internal segmentation of a company, f.e. sales department. |
```

Comment:
I assume "f.e." is "for example"? I would prefer the standard "e.g.", I don't think I've ever seen "f.e." before